internal/monitors/prometheusexporter/prometheus.go

```diff
@@ -56,6 +56,9 @@ type Config struct {
 	// Path to the client TLS key to use for TLS required connections
 	ClientKeyPath string `yaml:"clientKeyPath"`
 
+	// Number of seconds before timing out while requesting data from target host.
+	HTTPTimeout time.Duration `yaml:"httpTimeout" default:"10"`
```

Comment:
time.Duration has units of nanoseconds not seconds.

You can do a default of `10s` and I think it will parse it from yaml properly (see other config options that are time.Duration).

Reply:
Ah yeah I was later multiplying it by time.Second so it was wrong.. but worked. :P Will update.